main.py

```diff
@@ -376,20 +376,20 @@ def emailLogForCourseID(courseID, recipients):
         return
 
     try:
-        READ_BINARY_MODE = 'rb'
-        logfile = open(getCourseLogFilePath(courseID), mode=READ_BINARY_MODE)
+        logfile = open(getCourseLogFilePath(courseID))
         logContent = logfile.read()
         logfile.close()
     except Exception as exception:
         logger.warning('Exception while trying to read logfile for course {courseID}: {exception}'
```

Comment:
Do we have a monitoring method in place now, so that the dev team will get a warning message whenever there is an error with kartograafr cron job execution?

No one will look into the log file for warning messages.

Reply:
Will capture this in a separate issue for future development